modules/intensity_transform/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+set(the_description "Intensity transformations")
+ocv_define_module(intensity_transform opencv_core opencv_imgcodecs WRAP python)
```

Comment:
> imgcodecs

Looks like this dependency is not used in module itself ("samples" has own extra dependencies list).
Please remove this from here and `precomp.hpp`